### DISCUSSION TITLE
Issues to close (read more)

### DISCUSSION THREAD

**Discussion by quick007** [Minehut/Meta#385] [General] [answered]
## Overview

Hi, I went ahead and looked through a lot of the issues to find ones that should be closed in my opinion. If you see something here that you don't think should be closed, feel free to comment. If you want to add something to the list, comment as well :smiley:

## Features that need to be closed
These are more of features that are highly disliked, but reports, and other general stuff.

- [ ] #30 - Trent and the devs have said multiple times that the forums won't be getting a revamp because invasion is what they want to stick with
- [ ] #42 - I'm 99% sure that this has been implemented already
- [x] #46 - Bug has been fixed
- [x] #62 - Another issue (#342), opened by Trent is working on addressing larger issues with the color scheme
- [x] #123 - We now have around the block, even if it's trash...

## Features that could be closed just bc of feasibility/likes
The main goal of this category is dead threads and stuff. I know people will probably say stuff like "oh but this could be cool", I think that it's not worth it to just leave the stuff there. I did make this category because they aren't what I would call definite needs to be closed, so keep that in mind as you read these.

- [ ] #16 - Has no likes/dislikes and seems generally left unregarded
- [ ] #25 - Facto seems preoccupied with other stuff, and GitHub already has a half-decent embed: https://ibb.co/zSW8zCQ
- [ ] #27 - doesn't seem like a high priority bug (it's not planned on the projects tab), and honestly isn't a big feature that's needed.
- [ ] #36 - Low feedback, an unneeded QoL feature, I don't see a good reason for this.
- [ ] #44 - Not needed
- [ ] #47 - I haven't seen this bug in forever, look like it was packed. If not, relabel the issue as a bug please :smily:
- [ ] #64 - This can already be accomplished by just setting the plan to "free" and waiting for it to expire.
- [ ] #144 - Minehut already has a basics playlist, that should probably help new user

---

*I don't have enough time to look through the rest, this is like half of them so that's cool.*

**Comment by MiginsMC** [comment]
In regards to #25 I was actually making that, but I got super busy with exams and all that stuff. I was almost done so in a week or so I might look back into finishing it if you'd like.

**Comment by jellz** [comment]
>  #25 - Facto seems preoccupied with other stuff, and GitHub already has a half-decent embed: https://ibb.co/zSW8zCQ

There is an embed for GitHub links but the purpose of the issue is to embed the issue when only the hashtag and number is sent (e.g. `#120`), not the URL.

**Comment by Sniped** [comment, marked as answer]
> #42 - I'm 99% sure that this has been implemented already

Color codes still are counted when changing the server MOTD

> Features that could be closed just bc of feasibility/likes

Why close any of these issues for that reasoning?

> #16 - Has no likes/dislikes and seems generally left unregarded

I believe that rather than closing this suggestion, we could just bring light to the suggestion instead for further feedback.

> #25 - Facto seems preoccupied with other stuff, and GitHub already has a half-decent embed: https://ibb.co/zSW8zCQ

While I might be preoccupied with other stuff, other people can definitely contribute to the Minehut bot repo just as I do simply by making a pull request. 

> #27 - doesn't seem like a high priority bug (it's not planned on the projects tab), and honestly isn't a big feature that's needed.

Just like #16, I believe we should be bringing light to these issues rather than just closing them because they haven't gotten much attention.

> #36 - Low feedback, an unneeded QoL feature, I don't see a good reason for this.

I don't see any reason to not add this in the future, it would be a nice quality of life feature for people that have `.` and `_` in their world names.

> #44 - Not needed

Is there any reason to close this other than just "not needed"?

> #64 - This can already be accomplished by just setting the plan to "free" and waiting for it to expire.

This original user that commented this also stated that it would be nice to have an option to just cancel renewing and switch back to the free plan when it expires, which is what I believe OP was suggesting to begin with.

> #144 - Minehut already has a basics playlist, that should probably help new user

Sure, there's already some resources that help the user out, but is there a reason to completely close it due to this? It would definitely be a nice feature to have.

**Reply by Sniped** [reply]
Marking this as the answer to this as I've provided explanation for issues that I didn't already close. If you have any other issues you think should be closed feel free to put them here :)

**Comment by theminecoder** [comment]
Converted this to a discussion to remove it out of the issue tracker

**Reply by quick007** [reply]
> Converted this to a discussion to remove it out of the issue tracker

woah new feature

**Reply by jellz** [reply]
> woah new feature

It was announced nearly 5 months ago
